draft-ietf-acme-acme.md

```diff
@@ -1944,11 +1932,13 @@ for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
 ".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
```

Comment:
Let's just leave untouched the two bits that use normative language, and break out those fixes into their own PR so people can look a little more closely at them. This PR is pretty big.